packages/client/hmi-client/src/workflow/ops/calibrate-ciemss/tera-calibrate-ciemss.vue

```diff
@@ -234,6 +238,12 @@ const modelConfig = ref<ModelConfiguration>();
 
 const modelConfigId = computed<string | undefined>(() => props.node.inputs[0]?.value?.[0]);
 const datasetId = computed<string | undefined>(() => props.node.inputs[1]?.value?.[0]);
+const cancelRunId = computed(() => {
+	if (props.node.state.inProgressForecastId !== '') return props.node.state.inProgressForecastId;
+	if (props.node.state.inProgressCalibrationId !== '')
+		return props.node.state.inProgressCalibrationId;
+	return '';
+});
```

Comment:
The the majority of your `cancelRunid` function can be simplified somewhat:
- You don't need to repeat `props.node.state` so many times times, it makes the code harder to read
- isn't this effectively just `return state. inProgressForecastId  || state inProgressCalibrationId` ?

Reply:
I will rewrite this to be cleaner